h/queue.py

```diff
@@ -17,7 +17,14 @@ def publish(self, topic, data):
         return self.client.publish(topic, data)
 
 
-def get_reader(settings, topic, channel):
+def extra_context_data(req, topic=''):
```

Comment:
The request object should usually be called `request` in full, for the sake of consistency with other APIs that have a similar structure.